blocked-edges/4.7.24.yaml

```diff
@@ -0,0 +1,3 @@
+to: 4.7.24
+from: 4\.7\..*
+# SDN is down and kube-apiserver cannot reach aggregated apiserver: no route to host, SDN Packet loss resulting in service unavailability.  https://bugzilla.redhat.com/show_bug.cgi?id=1987108
```

Comment:
We should change the bug description to `Networking issue with vSphere clusters running HW14 and later`

Reply:
Changes the bug description!